loadgen/test_settings_internal.h

```diff
@@ -68,6 +68,9 @@ struct TestSettingsInternal {
   uint64_t qsl_rng_seed;
   uint64_t sample_index_rng_seed;
   uint64_t schedule_rng_seed;
+
+  double latency_percentile_multistream;
+  double latency_percentile_server;
```

Comment:
For TestSettingsInternal consistency, we should have a single latency_percentile for all scenarios.

Reply:
I will fix these.